AppleScript/AppleScript.sublime-syntax

```diff
@@ -552,25 +553,61 @@ contexts:
         # is / isn't
           is (?: n't |\s+ not )? (?:\s+
           (?: equal (?:\s+ to)?
-          | (?: less | greater )\s+ than (?:\s+ or \s+ equal (?:\s+ to )?)?
-          | in | contained \s+ by
+          | (?: less | greater )\s+ than (?:\s+ or\s+equal (?:\s+ to )?)?
+          | in | contained\s+by
```

Comment:
I spaced out the `\s+` to give space for space-collapsing merge strategies to do the right thing.